packages/core/core/src/services/document-service/transform/__tests__/utils.ts

```diff
@@ -0,0 +1,47 @@
+import { Schema, Common } from '@strapi/types';
+
+const createSchemaFromAttributes = (
+  uid: Common.UID.ContentType,
+  attributes: Schema.Attributes
+): Schema.ContentType => {
+  return {
+    uid,
+    info: { displayName: 'Test', singularName: 'test', pluralName: 'tests' },
+    kind: 'collectionType',
+    modelName: uid,
+    globalId: uid,
+    modelType: 'contentType',
+    attributes,
+  };
+};
+
+export const CATEGORY_UID = 'api::category.category' as Common.UID.ContentType;
+export const PRODUCT_UID = 'api::product.product' as Common.UID.ContentType;
+
+export const models: Record<string, Schema.ContentType> = {
+  [CATEGORY_UID]: createSchemaFromAttributes(CATEGORY_UID, {
+    name: {
+      type: 'string',
+    },
+  }),
+  [PRODUCT_UID]: createSchemaFromAttributes(PRODUCT_UID, {
+    name: {
+      type: 'string',
+    },
+    categories: {
```

Comment:
Or maybe not? They might not be relevant here since we don't expose IDs for things like components and dynamic zones, but we need to investigate how it will work for custom morph relations in schemas and if we need to support that.

Reply:
I have created a ticket to handle morph relations, and that will be prioritized in the team